### PR TITLE
Provision blockchain node

### PR DESCRIPTION
**Pull request by DerekCrosson**
Work in progress

### CODE REVIEW

File: Task-4-blockchain-node/modules/blockchain_node/variables.tf [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+variable "node_count" {
+  default = 1
+}
+
+variable "name" {
+  default = "node"
+}
+
+variable "image" {
+  default = "ami-0e6273fe5a9a1ad93"
```
Comment (DerekCrosson) [inline Task-4-blockchain-node/modules/blockchain_node/variables.tf]: Not 100% sure what the image here should be. We can change it to a different one

File: Task-4-blockchain-node/modules/blockchain_node/variables.tf [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+variable "zone" {
+  default = "eu-west-1c"
+}
+
+variable "machine_type" {
+  default = "m4.large"
+}
+
+variable "public_key" {
+  default = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCzHVMV5vSGeoCZD9gzdPCpSkJcSH3b62mvZ0ZaQixkHu6SSCFdflA8CF4XKeqV/s4Jl+++8xeThNm8tMUT1nYNXlK0eCYmTZID/B75aI4aThXavzEJAxQTQ+1rl1Iiz8ydnFF/zexLOSonDNOZNbMGJylPPk3Wjgenpa1ezHmiZCNck+41/iNpTsrkel74CPmzuD6HWed4NCCZUnGAyHNAa3bXWBu0Uc6QsjCTFn5GXNg2SC1cgZTWY5Eli/HM1nQjt/xUHqqBgrotGRxoAwvWcUx4nlapCHmoftUw5akH20wnMx+AjovE27wM74fJNdAFtmw5HTRaS4RgVyFgy2v7 derek@Dereks-Air.Dlink"
```
Comment (DerekCrosson) [inline Task-4-blockchain-node/modules/blockchain_node/variables.tf]: We will need to generate a key and store it in Github settings maybe? I just hard coded my personal one for testing

File: Task-4-blockchain-node/modules/blockchain_node/variables.tf [hunk 3]
```diff
@@ -15,7 +15,7 @@ variable "machine_type" {
 }
 
 variable "public_key" {
-  default = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCzHVMV5vSGeoCZD9gzdPCpSkJcSH3b62mvZ0ZaQixkHu6SSCFdflA8CF4XKeqV/s4Jl+++8xeThNm8tMUT1nYNXlK0eCYmTZID/B75aI4aThXavzEJAxQTQ+1rl1Iiz8ydnFF/zexLOSonDNOZNbMGJylPPk3Wjgenpa1ezHmiZCNck+41/iNpTsrkel74CPmzuD6HWed4NCCZUnGAyHNAa3bXWBu0Uc6QsjCTFn5GXNg2SC1cgZTWY5Eli/HM1nQjt/xUHqqBgrotGRxoAwvWcUx4nlapCHmoftUw5akH20wnMx+AjovE27wM74fJNdAFtmw5HTRaS4RgVyFgy2v7 derek@Dereks-Air.Dlink"
```
Comment (DerekCrosson) [inline Task-4-blockchain-node/modules/blockchain_node/variables.tf]: Note to self: regenerated and replaced on all cloud platforms

File: Task-4-blockchain-node/modules/remote_state/templates/remote-state.tpl [hunk 4]
```diff
@@ -0,0 +1,7 @@
+terraform {
```
Comment (praizz) [inline Task-4-blockchain-node/modules/remote_state/templates/remote-state.tpl]: We doo not need this

Comment (DerekCrosson) [reply Task-4-blockchain-node/modules/remote_state/templates/remote-state.tpl]: @praizz just the template or the whole remote state module?

**Comment by DerekCrosson** [comment]
Assignment change from group to individual

**Pull request closed without merging**